perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ServicesManagerBlImpl.java

```diff
@@ -980,7 +981,11 @@ public void setPerunBl(PerunBl perunBl) {
 				Destination destination = new Destination();
 				destination.setDestination(host.getHostname());
 				destination.setType(Destination.DESTINATIONHOSTTYPE);
-				destinations.add(this.addDestination(perunSession, service, facility, destination));
+				try {
+					destinations.add(this.addDestination(perunSession, service, facility, destination));
+				} catch (InvalidDestinationException e) {
+					throw new ConsistencyErrorException("Destination created out of hostname has invalid name.", e);
```

Comment:
Not sure, if we should cover this up. `ConsistencyErrorException` is not expected to be fixed by user, but should be reported to support. In this case, user should be able to fix it by simply fixing Host name, so I would prefer simply throwing `InvalidDestinationException` again with updated text, that invalid "value" was taken from host.

Same for the other changed method.

Reply:
But hosts are now checked when creating by the same pattern as destination of host type, so if we were able to create host with this hostname, but unable to create destination it is weird and it shouldn't happen...

Reply:
You are probably right. I just thought about existing Hosts, which were created before the check was deployed, but from this point of view you are right. It shouldn't happen. So keep it as it is.